hpi-schul-cloud/schulcloud-server
# BC-3135 Delete leave team permission form teammember

# Description

Deleted permission leave_team in temmember (atendee) role

QA notes:
Ticket tested all details attached. Please decide what the end result should look like. At the moment "Leave team" option does not exist see attachment in jira. Ticket did not pass the tests.

https://ticketsystem.dbildungscloud.de/browse/BC-3135



## Links to Tickets or other pull requests

https://ticketsystem.dbildungscloud.de/browse/BC-3135

## Changes

Changes made with migration and db setup files.

## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
